package.json

```diff
@@ -3,6 +3,7 @@
 	"title": "WooCommerce Monorepo",
 	"description": "Monorepo for the WooCommerce ecosystem",
 	"homepage": "https://woocommerce.com/",
+	"packageManager": "pnpm@^9.1.0",
```

Comment:
`pnpm/action-setup` will pick it up, giving us opportunity to reduce complexity a bit.